src/main/resources/db/v3__create_reservation_table.sql

```diff
@@ -0,0 +1,10 @@
+create table reservation (
+       id bigserial not null,
+        number_of_seats integer,
+        canceled boolean,
+        projection_id bigint not null,
+        user_id bigint not null,
+        primary key (id)
+    );
+alter table if exists reservation add constraint fk_projection_id  foreign key (projection_id) references projection (id);
+alter table if exists reservation add constraint fk_user_id foreign key (user_id) references "user" (id);
```

Comment:
You can write FK like this:

`projection_id bigint not null references projection`

It will auto-generate constraint name, but is more readable